src/include/distributed/resource_lock.h

```diff
@@ -39,6 +39,7 @@ typedef enum AdvisoryLocktagClass
 	ADV_LOCKTAG_CLASS_CITUS_REBALANCE_COLOCATION = 7,
 	ADV_LOCKTAG_CLASS_CITUS_COLOCATED_SHARDS_METADATA = 8,
 	ADV_LOCKTAG_CLASS_CITUS_OPERATIONS = 9,
+	ADV_LOCKTAG_CLASS_CITUS_PLACEMENT_CLEANUP = 11,
```

Comment:
I didn't wanna change the existing numbers here "ADV_LOCKTAG_CLASS_COLUMNAR_STRIPE_RESERVATION". I guess there is no harm in assigning an out-of-order number here, does this look odd?

Reply:
Also we can possibly include this under CitusOperations?

Reply:
Maybe just change the numbers?